pkg/model/BUILD.bazel

```diff
@@ -55,6 +55,7 @@ go_library(
         "deployment.go",
         "deployment_chain.go",
         "docs.go",
+        "environment.pb.go",
```

Comment:
Please remove this since it is no longer needed.
(I think you have to delete it at your local as well.)